### PR TITLE
Remove orquesta

### PR DESCRIPTION
**Pull request by anish-mudaraddi**
### Description:
This feature branch removes orquesta workflows entirely 

### Special Notes:

---

### Submitter:

Have you (where applicable):

* [ ] Added unit tests?
* [ ] Checked the latest commit runs on Dev?
* [ ] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [ ] Place non-StackStorm code into the `lib` directory?
* [ ] Have unit tests for the action/sensor and `lib` layers?
* [ ] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

File: tests/lib/openstack_api/test_openstack_roles.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            project_identifier="bar",
+            role_identifier="  ",
+        ),
+    )
+    mock_conn.identity.assign_project_role_to_user.assert_not_called()
+
+
+def test_assign_roles_successful():
+    """
+    Tests that an exception is thrown if the specified user is missing
```
Comment (gmatthews20) [inline tests/lib/openstack_api/test_openstack_roles.py:77]: Tests that assignment is successful

File: tests/lib/openstack_api/test_openstack_roles.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            project_identifier="bar",
+            role_identifier="  ",
+        ),
+    )
+    mock_conn.identity.unassign_project_role_from_user.assert_not_called()
+
+
+def test_remove_roles_successful():
+    """
+    Tests that an exception is thrown if the specified user is missing
```
Comment (gmatthews20) [inline tests/lib/openstack_api/test_openstack_roles.py:160]: Test is successful

File: tests/lib/workflows/test_project_actions.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    # setup input values
+    mock_conn = MagicMock()
+    mock_project_identifier = "project-id"
+
+    mock_conn.identity.find_project.return_value = {
+        "id": "project-id",
+        "name": "project-name",
+        "description": "project-description",
+        "email": "example@example.com",
+    }
```
Comment (gmatthews20) [inline tests/lib/workflows/test_project_actions.py:51]: Should this also have "tags" as the above test has

**Review by gmatthews20 — DISMISSED** [review]

**Comment by anish-mudaraddi** [comment]
This has been tested and has been confirmed to work on dev

File: .pylintrc [hunk 4]
```diff
@@ -10,4 +10,4 @@ max-line-length=120
 # R0801: Similar lines - Imports and method signatures will flag this, such as forwarding action args
 # C0116: Missing method docstring - Adds too much noise
```
Comment (DavidFair) [inline .pylintrc:12]: We're missing the list of disables and why in the comments above for these new ones

**Review by DavidFair — APPROVED** [review]

**Review by DavidFair — APPROVED** [review]

**Review by gmatthews20 — APPROVED** [review]

**Pull request merged**